src/core/effects/qgsimageoperation.cpp

```diff
@@ -691,9 +657,7 @@ void QgsImageOperation::GaussianBlurOperation::operator()( QgsImageOperation::Im
       sourceRef = sourceFirstLine;
       destRef = reinterpret_cast< QRgb * >( outputLineRef );
       for ( int x = 0; x < width; ++x, ++destRef, sourceRef += 4 )
-      {
-        *destRef = gaussianBlurVertical( y, sourceRef, sourceBpl, height );
-      }
+        * destRef = gaussianBlurVertical( y, sourceRef, sourceBpl, height );
```

Comment:
Hmmm... the asterisk is used to dereference in this situation, according to my preference it should be variable-aligned here.

Not sure if astyle is able to properly disambiguate.